boot/zephyr/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -31,7 +31,7 @@ CONFIG_NORDIC_QSPI_NOR=y
 CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
 CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
 
-# Required by USB and QSPI
+# Required by USB
```

Comment:
in own commit with `nrf-squash! [nrf noup] boot: nrf53-specific customizations` in commit message